.storybook/main.ts

```diff
@@ -1,16 +1,26 @@
 import type { StorybookConfig } from '@storybook/react-vite';
 
 const config: StorybookConfig = {
-  stories: ['../src/**/*.mdx', '../src/**/*.stories.@(js|jsx|mjs|ts|tsx)'],
+  stories: [
+    '../src/**/*.mdx',
+    '../src/**/*.stories.@(js|jsx|mjs|ts|tsx)'
+  ],
   addons: [
     '@chromatic-com/storybook',
-    '@storybook/addon-docs',
+    '@storybook/addon-essentials',
     '@storybook/addon-a11y',
-    '@storybook/addon-vitest',
+    '@storybook/addon-interactions',
   ],
   framework: {
     name: '@storybook/react-vite',
-    options: {},
+    options: {}
+  },
+  docs: {
+    autodocs: true
+  },
+  core: {
```

Comment:
The Storybook config enables crash reports via `core.enableCrashReports: true`. Sending crash reports may have privacy, telemetry, or corporate policy implications. This setting should be intentionally enabled and documented.

**Sugerencia:**
```typescript
Document the reason for enabling crash reports and confirm it follows org policy. If not required, set `enableCrashReports: false`.
```